scripts/SANS/sans/gui_logic/presenter/run_tab_presenter.py

```diff
@@ -423,6 +427,19 @@ def on_user_file_load(self):
             traceback.print_exc()
             self._on_user_file_load_failure(other_error, "Unknown error in loading user file.", use_error_name=True)
 
+    # When performing merged reduction, if both the scale option and the plot result option are selected,
+    # it causes an issue. The agreed temporary fix is to hide the plot result checkbox by default since
+    # the functionality is rarely used. Users can enable the visibility of the plot result checkbox
+    # from their properties file if needed. This experiment will help us determine whether to
+    # permanently hide the checkbox or fix the underlying issue.
+    def hide_or_show_plot_results_checkbox_based_on_user_properties(self):
+        visibility = ConfigService.getString("sans.isis_sans.plotResults")
```

Comment:
For documentation like this, it's better to use the [docstrings provided by python](https://peps.python.org/pep-0257/) (the first line is a summary, then a blank line, then more context for the method):
```suggestion
    def hide_or_show_plot_results_checkbox_based_on_user_properties(self):
        """Hide the plot results checkbox if it has not been explicitly enabled in the user properties file.
       
        When performing merged reduction, if both the scale option and the plot result option are selected,
        it causes an issue. The agreed temporary fix is to hide the plot result checkbox by default since
        the functionality is rarely used. Users can enable the visibility of the plot result checkbox
        from their properties file if needed. This experiment will help us determine whether to
        permanently hide the checkbox or fix the underlying issue.
        """
        visibility = ConfigService.getString("sans.isis_sans.plotResults")
```